controllers/articles.controller.js

```diff
@@ -44,8 +45,23 @@ exports.postComment = (req, res, next) => {
   ];
 
   Promise.all(promises)
-    .then((promisesResults) => {
-      res.status(201).send({ comment: promisesResults[1] });
+    .then(([_, comment]) => {
+      res.status(201).send({ comment });
+    })
+    .catch(next);
+};
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+  const promises = [
+    fetchArticle(article_id),
+    updateArticle(article_id, inc_votes),
+  ];
+
+  Promise.all(promises)
+    .then(([_, article]) => {
```

Comment:
This is nice de-structuring of an array, but alternatively if you switch the positions of the promises in the original promises array you wont need to have a blank first elem and you can just de-structure the first element which would be the article you want